docs/cloud-native-security/benchmark-rules.asciidoc

```diff
@@ -1,2 +1,7 @@
 [[benchmark-rules]]
 = Benchmark rules
+The Benchmark Integrations page helps you view and manage the CSP benchmark rules that are active for each of your <<kspm, KSPM integrations>>. Benchmark rules define security best practices for Kubernetes configuration, and form the basis of <<kspm, Findings>>.
```

Comment:
It'd probably be good to spell out CSP and KSPM on these first references. Also the reference to "Findings" seems a little obscure, esp if this is the first time learning about it -- maybe a little more context:
```suggestion
The Benchmark Integrations page helps you view and manage the cloud security posture (CSP) benchmark rules that are active for each of your <<kspm, Kubernetes security posture management (KSPM) integrations>>. Benchmark rules define security best practices for Kubernetes configuration, and form the basis of the results on the <<kspm, Findings page>>.
```